test/views/SignedOutViewSpec.scala

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package views
+
+import views.behaviours.ViewBehaviours
+import views.html.WeSignedYouOutPageView
+
+class SignedOutViewSpec extends ViewBehaviours {
+
+  "Session Expired view" must {
+    "Show sign out page" in {
+
+      val application = applicationBuilder().build()
+
+      val view = application.injector.instanceOf[WeSignedYouOutPageView]
+
+      val applyView = view.apply()(fakeRequest, messages)
+
+      val doc = asDocument(applyView)
+
+      doc.title() mustBe "For your security, this service has been reset"
```

Comment:
This test should be: 
`doc.title() mustBe "For your security, we signed you out"`